packages/app-store/ee/routing-forms/playwright/tests/basic.e2e.ts

```diff
@@ -215,13 +215,22 @@ test.describe("Routing Forms", () => {
           resolve(body);
         });
       });
+      const csvRows = csv.trim().split("\n");
+      const csvHeaderRow = csvRows[0];
+      expect(csvHeaderRow).toEqual("Test field,Multi Select,Submission Time");
 
-      expect(csv.trim()).toEqual(
-        `
-"Test field :=> event-routing"
-"Test field :=> external-redirect"
-"Test field :=> custom-page"`.trim()
-      );
+      const firstResponseCells = csvRows[1].split(",");
+      const secondResponseCells = csvRows[2].split(",");
+      const thirdResponseCells = csvRows[3].split(",");
+
+      expect(firstResponseCells.slice(0, -1).join(",")).toEqual("event-routing,");
+      expect(new Date(firstResponseCells.at(-1)).getDay()).toEqual(new Date().getDay());
```

Comment:
Can't match exact time of form submission(as it is variable). Check just the date for now.